events/elections/2019/README.md

```diff
@@ -0,0 +1,155 @@
+# 2019 VOTERS GUIDE - KUBERNETES STEERING COMMITTEE ELECTION
+
+## Purpose
+
+The role of this election is to fill out the four (4) seats due for
+reelection this year on the [Kubernetes Steering Committee]. Each elected
+member will serve a two (2) year term.
+
+## Background
+
+This election will shape the future of Kubernetes as a community and project.
+While SIGs and WGs help shape the technical direction of the project, the
+[Steering Committee Charter] covers the health of the project and community
+as a whole. Some direct responsibilities of steering members to consider as you are deciding
+whether to run or who to vote for:
+
+* Through the chartering review process, delegate ownership of, responsibility for
+  and authority over areas of the project to specific entities.
+* Define, evolve, and defend the non-technical vision / mission and the values
+  of the project.
+* Charter and refine policy for defining new community groups.
+  and establish transparency and accountability policies for such groups
+* Define and evolve project and group governance
+  structures and policies.
+* Act as a final non-technical escalation point for any Kubernetes repository
+* Request funds and other support from the CNCF (e.g. marketing, press, etc.)
+* Define and enforce requirements for community groups to be in good standing 
+  such as having an approved charter.
+
+For more context, please see the [current steering committee backlog] or a
+previous [governance meeting video] which led to this whole process.
+
+## Eligibility
+
+Please refer to the [Steering Committee Election Charter] for:
+
+- [eligibility for candidacy]
+- [eligibility for voting] and [eligible voters] list (not in charter)
+
+### Schedule
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| August 23    | Announcement of Election |
```

Comment:
Reminder to update the dates and times

Reply:
21 August - Announce Election - first thing in the morning EST, this is
just sending an email
11 Sept - All bios and voting exemptions due - 5pm PST
18 Sept - Voting begins - "Morning of" to account for CIVS setup
2 October - Voting ends - 5pm PST
3 October - Results published on blog/community meeting - Normal
community meeting time, blog published right after.